### PR TITLE
Have gradle include all jars, not just classes.jar

### LINKED ISSUE #225: No longer completes classes on Android
## Actual behavior (Required!)
When attempting to complete an import in an Android project, Javacomplete says "packages still empty, try later. indexing..." even on non-android imports

Also related behavior, when completing within a class that extends an Android component there are missing completions when completing ```this``` or ```super```

## Expected behavior (Required!)
Should give imports and complete android related classes

## The steps to reproduce actual behavior (Required!)
 1. Create a basic Android Studio project or use [Plaid](https://github.com/nickbutcher/plaid)
 2. Open the MainActivity.java or other file that extends an Activity
 3. Attempt to complete an android import
 4. ```:messages``` will show that the app is indexing, even after several minutes
 5. Move to class body and attempt completing ```this``` or ```super``` and receive no results or ones that are missing

## Environment (Required!)
 * OS: Archlinux
 * Neovim version: Neovim 1.3.0-68-gd19af41

## Q&A
- [x] Yes, I tried minimal .vimrc configuration.
- [x] Yes, I have enabled logs with `JCdebugEnableLogs` and can put here content of `JCdebugGetLogContent` command, if you need.
- [x] Even, if you wish, I can set `g:JavaComplete_JavaviDebug` to `1`, than set `g:JavaComplete_JavaviLogfileDirectory`, and put here server logs, too.
- For clarification my minimal vim was simply setting runtimepath to the location of vim-javacomplete2 and setting the omnifunc

## Additional information

I'm opening this issue to solicit insight from others and begin tracking down what exactly is causing this. I'm happy to provide additional information, and I suspect that this is not Android specific but I'm only getting these issues within Android projects right now

### PR DESCRIPTION
**Pull request by DonnieWest**
Fix #225

### CODE REVIEW

**Comment by artur-shaik** [comment]
Great, thank you!

File: classpath.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ task classpath << {
     def listFiles = new File(exploded)
     if (listFiles.exists()) {
       listFiles.eachFileRecurse(){ file ->
-        if (file.name.endsWith("classes.jar")){
+        if (file.name.endsWith(".jar")){
```
Comment (wsdjeg) [inline classpath.gradle]: really can not understand why need change `classes.jar` to `.jar`,here line8 listFiles is in the buildDir?so the `.jar` is generated by gradle?

Comment (DonnieWest) [inline classpath.gradle]: In most Gradle projects (including Android) the build directory is used as a sort of temporary storage area for files that Gradle has processed. To get java to compile the resulting files into .class files, they are bundled into .jar files and were typically called "classes.jar." It is not a requirement though to name them "classes.jar" and that is why this change is needed - we were missing internal_impl-23.1.1.jar and in the future there may be more jars outside of the "classes.jar" convention

At least that's how I understand it

Comment (wsdjeg) [inline classpath.gradle]: thanks :smile:

**Comment by wsdjeg** [comment]
@DonnieWest 
nice works

**Pull request merged**